input/rmatch.in

```diff
@@ -1,94 +1,94 @@
 /
 &DELTAC_LIST
-    tid=3,5,6,4
-    deltac_bin_sol=f
-    deltac_out_sol=f
+    tid=3,5,6,4          ! Something to do with which solution indexes are used in computing combined solutions. Always 3,5,6,4 in examples.
+    deltac_bin_sol=f     ! Write deltac solutions to binary files for xdraw
+    deltac_out_sol=f     ! Write deltac solutions to ascii files
 
-    rescale=t
-    pfac=0.1
-    xfac=10.0
-    gal_method="normal"
-    noexp=t
-    nx=128
-    nq=5
-    deltac_tol=1e-11
-    dx1dx2_flag=f
-    dx1=0.001
-    dx2=0.002
-    restore_ul=t
-    restore_uh=t
-    restore_us=t
+    rescale=t            ! Rescale x during asymptotic matching for better numerical behavior
+    pfac=0.1             ! Packing ratio? Should it be the same as in RDCON?
+    gal_method="normal"  ! Something to do with cell types used? Options are "normal", "resonant". "normal" in examples.
+    noexp=t              ! Unknown function. Always true in examples.
+    nx=128               ! The number of elements in each interval between two singular surfaces
+    nq=5                 ! The number of Gaussian points in each Galerkin element
+    deltac_tol=1e-11     ! Tolerance of lsode integration
+    dx1dx2_flag=f        ! Flag to include the special dx1 and dx2 treatments for resonant and extension element
+    dx1=0.001            ! The size of resonant element
+    dx2=0.002            ! The size of extension element
+    restore_ul=t         ! Include the Hermite solution in Galerkin solution
+    restore_uh=t         ! Include the small solution in Galerkin solution
+    restore_us=t         ! Include the larger solution in Galerkin solution
 
-    cutoff=20
+    cutoff=20            ! The number of elements include the large solution as the driving term
 
-    interp_np=5
-    order_pow=20
-    order_exp=20
+    interp_np=5          ! The number of interpretation points for outputting Galerkin solution
+    order_pow=20         ! The order of polynomial expansion of large solution in asymptotic matching
+    order_exp=20         ! The order of exponential expansion of small solution in asymptotic matching
 
-    diagnose_res=f
+    diagnose_res=f       ! Diagnose asymptotic solutions in ua.out/ua.bin
 
-    x0_ua=0.3
-    x1_ua=1
-    nx_ua=100
-    outt=3
+    x0_ua=0.3            ! Unknown function for diagnose_res
+    x1_ua=1              ! Unknown function for diagnose_res
+    nx_ua=100            ! Number of points for diagnose_res
+    outt=3               ! Unknown function for diagnose_res
 /
 &NYQUIST_INPUT
-    nyquist%flag=f
-    nyquist%ns=100
+    nyquist%flag=f       ! Make nyquist plot
+    nyquist%ns=100       ! Parameter for nyquist plot
 
-    nyquist%small=.01
-    nyquist%big=300
-    nyquist%narc=32
+    nyquist%small=.01    ! Unknown function for nyquist plot
+    nyquist%big=300      ! Unknown function for nyquist plot
+    nyquist%narc=32      ! Unknown function for nyquist plot
```

Comment:
How fine a grid in |Q| to search for roots of the dispersion relation (between nyquist%small and nyquist%big)